UpdatedImageProcessing/UpdatedImageProcessing/ShapeDetection/shape_classification.py

```diff
@@ -8,6 +8,7 @@
 import numpy
 
 CIRCLE_SCORE_THRESHOLD = 0.45 #needs to be optomized
+NOISE_SCORE_THRESHOLD = 0.6
 SQUARE_RECTANGLE_EIGEN_RATIO_THRESHOLD = 1.5 #not tested
 PENTAGON_STAR_CLUSTER_THRESHOLD = 8 #not tested
 OCTAGON_CROSS_CLUSTER_THRESHOLD = 10 #not tested
```

Comment:
[There should be spaces in between the `#` and the text of the inline comment, per PEP8 standards.](https://www.python.org/dev/peps/pep-0008/#inline-comments)

Reply:
Additionally, should these variables be moved to a `settings.py` file, @DarkmatterVale ?

Reply:
Yea I agree.